packages/yew/src/dom_bundle/bcomp.rs

```diff
@@ -393,20 +393,6 @@ mod tests {
         };
         assert_eq!(get_html(for_method, &root, &scope, &parent), expected_html);
     }
-
-    #[test]
-    fn component_node_ref_stays_none() {
-        let (root, scope, parent) = setup_parent();
-
-        let node_ref = NodeRef::default();
-        let elem = html! { <Comp ref={node_ref.clone()}></Comp> };
-        let (_, elem) = elem.attach(&root, &scope, &parent, NodeRef::default());
-        scheduler::start_now();
-        assert!(node_ref.get().is_none(), "components don't have node refs");
-        elem.detach(&root, &parent, false);
-        scheduler::start_now();
-        assert!(node_ref.get().is_none(), "components don't have node refs");
-    }
```

Comment:
Test removed because this is now a compile error, see component-fail.stderr above.